editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2645,7 +2651,12 @@ static void draw_indicator_bar(Control &surface, real_t fill, const Ref<Texture2
 	surface.draw_texture(icon, icon_pos);
 
 	// Draw text below the bar (for speed/zoom information).
-	surface.draw_string(font, Vector2(icon_pos.x, icon_pos.y + icon_size.y + 16 * EDSCALE), text, HALIGN_LEFT, -1.f, font_size);
+	Vector2 string_pos = Vector2(icon_pos.x, icon_pos.y + icon_size.y + 16 * EDSCALE);
+	surface.draw_string(font, string_pos, text, HALIGN_LEFT, -1.f, font_size);
+	if (p_zoom_limit_message) {
+		surface.draw_string(font, string_pos + Vector2(0, font_size * 2), "To zoom further, change the camera's", HALIGN_LEFT, -1.0f, font_size);
+		surface.draw_string(font, string_pos + Vector2(0, font_size * 3), "clipping planes (View -> Settings...)", HALIGN_LEFT, -1.0f, font_size);
```

Comment:
It would be great if the message string could be translatable, but I don't know how to handle the line break. Maybe add a `"\n"` in the string constant and then split it manually into separate lines?

So something like this, but with proper handling of corner cases.
```
String zoom_message = TTR("This is the \n zoom message"); 
Vector<String> lines = zoom_message.split("\n");
surface.draw_string(...., lines[0], ....);
surface.draw_string(...., lines[1], ....);
```
Though I'm not entirely sure how the translation web app would handle line breaks like this.

Reply:
Good point. Is there a way that we can tell translators to just put the line break somewhere in the middle? Do we want to support having zero line breaks or more than one (which would be more complicated than supporting just one)?

Reply:
We can add a `// TRANSLATORS:` comment but that's error prone, for proper localization this should likely be done as a `Label` with autowrap, not two hand-drawn strings IMO. Or kept on a single line.

Reply:
Could be done like the Sprite drag-and-drop contextual help:
https://github.com/godotengine/godot/blob/ba1adc384cc038e5b67fafce98050ce5441e0c45/editor/plugins/canvas_item_editor_plugin.cpp#L6677-L6684